### ISSUE TITLE
IteratorOfChars & IteratorOfBytes have too few contructors

### ISSUE THREAD

**Issue by andreoss** [yegor256/cactoos#1397]
It would be nice to have additional constructors for `IteratorOfChars` and `IteratorOfBytes` (and their Iterable counterparts) which accept `String`, Text` or `Bytes` as a parameter.
Thus, the following code
```                            
   new IteratorOfChars(
         text.asString().toCharArray()
    )
```
could be refactored as
```
   new IteratorOfChars(text)
```

**Comment by 0crat** [comment]
@paulodamaso[/z](https://www.0crat.com/u/paulodamaso) please, pay attention to this issue

**Comment by paulodamaso** [comment]
@0crat in

**Comment by 0crat** [comment]
The job [#1397](https://github.com/yegor256/cactoos/issues/1397) assigned to @victornoel[/z](https://www.0crat.com/u/victornoel), here is [why](https://www.0crat.com/footprint/C63314D6Z/ee2d688f-38c1-437c-bf8b-0b4d9efdf318); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be no monetary reward for this job

**Comment by victornoel** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/yegor256/cactoos/issues/1397#issuecomment-641265977))

@victornoel The user @victornoel[/z](https://www.0crat.com/u/victornoel) resigned from [#1397](https://github.com/yegor256/cactoos/issues/1397), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/yegor256/cactoos/issues/1397#issuecomment-641265977))

@victornoel Job refused in 1 hours - no penalty, see [§6](http://www.zerocracy.com/policy.html#6)

**Comment by andreoss** [comment]
Implemented in #1398

**Comment by 0crat** [comment]
Job `gh:yegor256/cactoos#1397` is not assigned, can't get performer
